ccm_web/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "dev:server": "nodemon --config nmon_config.json server/app.ts",
-    "dev:client": "webpack serve --config webpack/webpack.dev.ts",
+    "dev:server": "nodemon --inspect=0.0.0.0:9229 --config nmon_config.json server/app.ts",
+    "dev:client": "webpack --watch --config webpack/webpack.dev.ts",
```

Comment:
with `webpack --watch` option I saw the `main-X.js` being created and I just changed the static mapping path to  `index.html`.

Reply:
This eliminated 2 things
1. Locally Running UI on port 4010 and server 4000. So it was as if both FE and BE running in one container with same port similar to Prod setup
2. LTIJS `dev` mode for local setup

Reply:
The Watch behavior is similar to Myla's. Sometime I feel generating `main-js` is bit slower and I need to click on terminal window to start. But I am not too concern about it. I think with Hot replacement module this could be better or I feel time to generate a new file is configurable option

Reply:
We may want to experiment with [`concurrently`](https://www.npmjs.com/package/concurrently) so we can see more easily which process is producing which output (`webpack` or the server). This can be done later.

Reply:
OK, I will look into it later. I will open an issue so that we have it in mind